src/app/containers/ComscoreAnalytics/index.test.jsx

```diff
@@ -42,71 +54,56 @@ ContextWrap.propTypes = {
   origin: string.isRequired,
   platform: string.isRequired,
   toggleState: shape({}),
+  personalisation: shape({}),
 };
 
 ContextWrap.defaultProps = {
   toggleState: defaultToggleState,
+  personalisation: defaultPersonalisation,
 };
 
 describe('Comscore Analytics Container', () => {
-  it('should return null when toggle is disabled', () => {
-    const toggleState = {
-      comscoreAnalytics: {
-        enabled: false,
-      },
-    };
-    const { container } = render(
-      <ContextWrap
-        platform="amp"
-        pageType="article"
-        origin="bbc.com"
-        toggleState={toggleState}
-      >
-        <ComscoreAnalytics />
-      </ContextWrap>,
-    );
+  describe('Assertions - AMP', () => {
+    it('should return null when toggle is disabled', () => {
+      const toggleState = {
+        comscoreAnalytics: {
+          enabled: false,
+        },
+      };
+      const { container } = render(
+        <ContextWrap
+          platform="amp"
+          pageType="article"
+          origin="bbc.com"
+          toggleState={toggleState}
+        >
+          <ComscoreAnalytics />
+        </ContextWrap>,
+      );
 
-    expect(container.firstChild).toBeNull();
+      expect(container).toBeEmptyDOMElement();
+    });
   });
 
-  it('should return null when toggle is enabled but on canonical', () => {
-    const toggleState = {
-      comscoreAnalytics: {
-        enabled: true,
-      },
-    };
-    const { container } = render(
-      <ContextWrap
-        platform="canonical"
-        pageType="article"
-        origin="bbc.com"
-        toggleState={toggleState}
-      >
-        <ComscoreAnalytics />
-      </ContextWrap>,
-    );
-
-    expect(container.firstChild).toBeNull();
-  });
-
-  it('should render comscore script when on amp and toggle is enabled', () => {
-    const toggleState = {
-      comscoreAnalytics: {
-        enabled: true,
-      },
-    };
-    const { container } = render(
-      <ContextWrap
-        platform="amp"
-        pageType="article"
-        origin="bbc.com"
-        toggleState={toggleState}
-      >
-        <ComscoreAnalytics />
-      </ContextWrap>,
-    );
+  describe('Assertions - Canonical', () => {
+    it('should return null when toggle is disabled', async () => {
```

Comment:
Thanks Khoa! Maybe just a couple of snapshot tests in here (one for AMP and one for Canonical) when the toggle is enabled, and then good to go

Reply:
I've moved the snapshots from /canonical/index.test.jsx and /amp/index.test.jsx to index.test.jsx. And I refactored the amp tests so that they are asserting the content of the component file. 

https://github.com/bbc/simorgh/pull/7391/commits/1eb3e6981d93702311ce6e3767c5f34bde71e220